src/Client.php

```diff
@@ -275,6 +276,10 @@ public function notifyException($throwable, callable $callback = null)
     {
         $report = Report::fromPHPThrowable($this->config, $throwable);
 
+        if ($throwable instanceof MetaDataAwareInterface) {
```

Comment:
This should probably be implemented by a middleware, rather than being baked into the client like this.

Reply:
I'm not that familiar with this SDK. But the middleware doesn't get any exceptions, does it?

Reply:
Yeh, that is correct. Maybe we should change the report object to keep the actual exception around, so people can grab it (by means of a method on the report object that returns the exception or null).